romana-install/roles/romana/install-agent/tasks/romana_from_github.yml

```diff
@@ -9,3 +9,9 @@
   command: install -o root -g root -m 0755 "/var/tmp/{{ item }}" "{{ romana_bin_dir }}/{{ item }}"
   with_items:
     - agent
+
+- name: Install hook 1
+  template: src="config/agent_proxy_add_policy.sh" dest="{{ romana_bin_dir }}/" mode=0777
```

Comment:
Are these hooks only applicable when installing from github?
Since they aren't actually templates, they can be `files` with a `copy` task to install them.

I'd prefer they went in `romana_lib_dir`, since that's where I'm dumping things that are dependencies of other programs, rather than executed "directly".